src/main/kotlin/com/demonwav/mcdev/i18n/inspections/TranslationInspection.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n.inspections
+
+import com.demonwav.mcdev.platform.mcp.McpModuleType
+import com.intellij.codeInspection.BaseJavaLocalInspectionTool
+import com.intellij.codeInspection.InspectionManager
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.psi.PsiElementVisitor
+import com.intellij.psi.PsiFile
+
+abstract class TranslationInspection : BaseJavaLocalInspectionTool() {
```

Comment:
This should extend `AbstractBaseJavaLocalInspectionTool` now, as `BaseJavaLocalInspectionTool` is now deprecated.